multicluster/test/e2e/fixtures.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Antrea Authors
+// Copyright 2022 Antrea Authors
```

Comment:
do we need to change the year here when it's just modification of the file?

Reply:
We probably don't... but I believe the e2e framework was authored in 2022 as well?